scripts/fetchDefinitions.js

```diff
@@ -17,6 +17,53 @@ fs.mkdirSync(typesDir);
 rimraf.sync(generatedDir);
 fs.mkdirSync(generatedDir);
 
+/**
+ * @hidden
+ * transforms the schema so RPC types are compatible with other methods from the polkadot api.
+ * @note imports are added into the generated files in the postProcessTypes script
+ */
+function transformSchema(schemaObj) {
+  let {
+    types: { ComplianceRequirementResult, Condition, TrustedIssuer },
+    rpc: { identity, compliance, asset },
+  } = schemaObj;
+  identity.getFilteredAuthorizations.params = identity.getFilteredAuthorizations.params.map(p => ({
+    ...p,
+    name: camelCase(p.name),
+  }));
+  identity.getFilteredAuthorizations.type = 'Vec<PolymeshPrimitivesAuthorization>';
+
+  compliance.canTransfer.params = compliance.canTransfer.params.map(p => ({
+    ...p,
+    name: camelCase(p.name),
+  }));
+  const newCompliance = mapKeys(ComplianceRequirementResult, (v, k) => camelCase(k));
+  schemaObj.types.ComplianceRequirementResult = newCompliance;
+
+  const newCondition = mapKeys(Condition, (v, k) => camelCase(k));
+  newCondition.issuers = 'Vec<PolymeshPrimitivesConditionTrustedIssuer>';
+  schemaObj.types.Condition = newCondition;
+
+  const newTrustedIssuer = mapKeys(TrustedIssuer, (v, k) => camelCase(k));
+  schemaObj.types.TrustedIssuer = newTrustedIssuer;
```

Comment:
I think this can be encapsulated into a reusable function. At least the `mapKeys` part

Reply:
Same for the params transforming part